### DISCUSSION TITLE
Question on Tutorial

### DISCUSSION THREAD

**Discussion by sc2936** [nloyfer/wgbs_tools#32] [Q&A]
After git clone,
The command prompt is telling me that 'wgbstools' is not recognized as an internal or external command, operable program or batch file.

What do I need to do to fix this?

**Comment by yonniejon** [comment]
Hi,
Did you run the installation procedure?
See here: https://github.com/nloyfer/wgbs_tools there is the quick installation section. Basically you need to run the setup.py script. Besides that you can add an alias to link wgbstools to INSTALL_DIR/wgbs_tools/src/python/wgbs_tools.py
